renew_certificate.sh

```diff
@@ -35,3 +35,32 @@ kill -9 $pid || true
 rm -rf tmp-webroot
 
 /etc/init.d/Qthttpd.sh start
+# Nextcloud
+# certificat
```

Comment:
please add a new line between the qthttpd.sh start command and your nextcloud stuff, followed by a single line comment "# nextcloud certificate installation".